docs/developers/guides/community/irys/irys-nfts.mdx

```diff
@@ -178,4 +178,4 @@ irys upload metadata.json \
 ## Mint the NFTs
 
 Now use your metadata URLs (in the format `https://gateway.irys.xyz/:txId`) to mint the NFTs
-using [the contract you deployed on Linea](/developers/quickstart/deploy-smart-contract).
+using [the contract you deployed on Linea](../../../../developers/quickstart/deploy-smart-contract/).
```

Comment:
Doesn't resolve. Should it be ../../../../developers/quickstart/deploy-smart-contract/index.mdx 
?